wrench/src/reftest.rs

```diff
@@ -40,18 +39,28 @@ pub fn parse_reftests<F>(filename: &str, mut runner: F)
         }
 
         let mut items = s.split_whitespace();
-        let kind = match items.next() {
-            Some("==") => ReftestOp::Equal,
-            Some("!=") => ReftestOp::NotEqual,
+
+        match items.next() {
+            Some("include") => {
+                let include = dir.join(items.next().unwrap());
+                parse_reftests(include.as_path(), runner);
+            }
+            Some(x) => {
+                let kind = match x {
+                    "==" => ReftestOp::Equal,
+                    "!=" => ReftestOp::NotEqual,
+                    _ => panic!(),
```

Comment:
While we're changing this, let's make this panic include a message saying unexpected match operator.